tools-label-table.py

```diff
@@ -14,7 +14,14 @@ def create_table():
             if label_len > 1:
                 rowspan = " rowspan=" + str(label_len)
 
-            table += "  <tr>\n   <td" + rowspan + "><a>" + encoding["name"] + "</a>"
+            if encoding["name"] != "windows-1252":
+                table += "  <tr><td" + rowspan + ">" + encoding["name"] + "\n"
+            else:
+                table += f"""  <tr>
+   <td{rowspan}>
+    <a>{encoding["name"]}</a>
+    <p class=note>See <a href="#note-latin1-ascii">below</a> for the relationship to historical
+    "Latin1" and "ASCII" concepts."""
```

Comment:
I suspect this is missing a newline at the end.

Reply:
I added one, and indeed the Python script output is more consistent now, but the main text doesn't match the Python script output: all the blank lines are omitted. Not sure what to do about that, if anything.

Reply:
I took a more detailed look and pushed a fixup. Should be okay to land now from my perspective.